src/app/views/media/components/AssetMarkdown.js

```diff
@@ -19,12 +19,24 @@ const AssetMarkdown = ({ asset, value, onChange }) => {
   const readOnly = !['ARTICLE', 'LESSON'].includes(asset.asset_type);
   return (
     <Card>
-      {readOnly && 
+      {readOnly ?
         <Alert severity="warning">
           <AlertTitle>You cannot manually update this asset</AlertTitle>
           This asset is in synch with github, only pulling from the github repository you will be able to update its markdown content.
         </Alert>
+        : asset.last_synch_at < asset.readme_url  ?
```

Comment:
you should compare `asset.last_synch_at` with `asset.readme_updated_at` instead of `asset.readme_url`